Makefile

```diff
@@ -44,11 +44,7 @@ endif
 	    $< >$@
 
 install: $(NAME).pc
-	@$(MAKE) -C src install prefix=$(DESTDIR)$(prefix) \
-		includedir=$(DESTDIR)$(includedir) \
-		libdir=$(DESTDIR)$(libdir) \
-		libdevdir=$(DESTDIR)$(libdevdir) \
-		relativelibdir=$(relativelibdir)
+	@$(MAKE) -C src install prefix=$(DESTDIR)$(prefix)
```

Comment:
The prefix only applies to these 3:
```
includedir ?= $(prefix)/include
libdir ?= $(prefix)/lib
libdevdir ?= $(prefix)/lib
```

You shouldn't get rid of the `relativelibdir` here. What do you think?

Reply:
> You shouldn't get rid of the `relativelibdir` here. What do you think?

that gets deduces from `$libdir` in `./configure`, it seems, although it's only used when `ENABLE_SHARED` in switched on and i don't see that check in `./configure`, maybe @stefano-garzarella can shed some light on this?

leaving it there for now.